src/Console/ConfigurationResolver.php

```diff
@@ -774,6 +757,24 @@ private function validateRules(array $rules)
 
             throw new InvalidConfigurationException(substr($message, 0, -2).'.');
         }
+
+        foreach ($fixers as $fixer) {
```

Comment:
just asking here - why moving it from one method to another ?

Reply:
Current implementation does not work if you pass `"some_deprecated_fixer" => false`. Maybe this should be extracted as a bugfix PR though.